shared/oval/logwatch_configured_splithosts.xml

```diff
@@ -3,10 +3,10 @@
     <metadata>
       <title>Ensure Logwatch SplitHosts Configured</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>Check if SplitHosts line in logwatch.conf is set appropriately.</description>
-      <reference source="MED" ref_id="20130819" ref_url="test_attestation" />
+      <reference source="galford" ref_id="20160120" ref_url="test_attestation" />
     </metadata>
     <criteria>
```

Comment:
Again. The pattern for ```SplitHosts``` is too strict (explicitly requires one space to be present at the beginning on the row). Will fix this in the future. ACK